environment.py

```diff
@@ -114,6 +129,14 @@ def findCellsInRadialRange(self, startX, startY, gridRange):
                     cellsInRange.append({"cell": self.grid[reflectedX][reflectedY], "distance": euclideanDistance})
         return cellsInRange
 
```

Comment:
The two `findCellsIn*Range()` functions return lists of dictionaries with `cell`s and `distance`s, so the `cell`s are extracted into a list for the adjacency neighborhood.